city.py

```diff
@@ -5,7 +5,7 @@
 
 
 @dataclass
-class Game:
+class City:
```

Comment:
Please, make base class for City and Effect.
It should contain fields like:
context,
name,
effects

Reply:
May be implement this later, not a critical